src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -52,7 +52,7 @@ def list_role_definitions(name=None, resource_group_name=None, resource_id=None,
 
 helps['role create'] = """
             type: command
-            short-summary: Create a custom role 
+            long-summary: Create a custom role 
```

Comment:
per our conversation, i will remove it. This command doesn't need summary since the name itself is pretty clear